games-strategy/0ad/0ad-0.27.0.ebuild

```diff
@@ -0,0 +1,234 @@
+# Copyright 2014-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_10 python3_11 python3_12 python3_13)
+WX_GTK_VER="3.2-gtk3"
+inherit desktop flag-o-matic python-any-r1 toolchain-funcs multiprocessing wxwidgets xdg
+
+if [[ ${PV} != 9999 && ${PV} != *_pre* ]]; then
+	VERIFY_SIG_METHOD=minisig
+	# The public key can be found upstream - last update was w/ alpha 26:
+	# https://gitea.wildfiregames.com/0ad/0ad/wiki/VerifyingYourDownloads
+	VERIFY_SIG_OPENPGP_KEY_PATH=${FILESDIR}/0ad-minisign.pub
+	inherit verify-sig
+fi
+
+DESCRIPTION="A free, real-time strategy game"
+HOMEPAGE="https://play0ad.com/"
+LICENSE="BitstreamVera CC-BY-SA-3.0 GPL-2 LGPL-2.1 LPPL-1.3c MIT ZLIB"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/0ad/0ad"
+	S="${WORKDIR}/${P}"
+elif [[ ${PV} == *_pre* ]]; then
+	ZEROAD_GIT_REVISION="c7d07d3979f969b969211a5e5748fa775f6768a7"
+	SRC_URI="https://github.com/0ad/0ad/archive/${ZEROAD_GIT_REVISION}.tar.gz -> ${P}.tar.gz"
+	S="${WORKDIR}/${PN}-${ZEROAD_GIT_REVISION}"
+elif [[ ${PV} == *_rc* ]]; then
+	MY_P="0ad-${PV/_/-}"
+	SRC_URI="
+		https://releases.wildfiregames.com/rc/${MY_P}-unix-build.tar.xz
+		https://releases.wildfiregames.com/rc/${MY_P}-unix-data.tar.xz
+		verify-sig? (
+			https://releases.wildfiregames.com/rc/${MY_P}-unix-build.tar.xz.minisig
+			https://releases.wildfiregames.com/rc/${MY_P}-unix-data.tar.xz.minisig
+		)
+	"
+	S="${WORKDIR}/${MY_P/-rc*/}"
+else
+	MY_P="0ad-${PV/_/-}"
+	SRC_URI="
+		https://releases.wildfiregames.com/${MY_P}-unix-build.tar.xz
+		https://releases.wildfiregames.com/${MY_P}-unix-data.tar.xz
+		verify-sig? (
+			https://releases.wildfiregames.com/${MY_P}-unix-build.tar.xz.minisig
+			https://releases.wildfiregames.com/${MY_P}-unix-data.tar.xz.minisig
+		)
+	"
+	S="${WORKDIR}/${MY_P}"
+fi
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="editor +lobby nvtt pch test"
+
+RESTRICT="!test? ( test )"
+CHECKREQS_DISK_BUILD="4000M" # 3842680 KiB (3.6 GiB) for alpha 27
+CHECKREQS_DISK_USR="3500M" # 3452564 KiB (3.2 GiB)
+
+BDEPEND="
+	>=dev-util/premake-5.0.0_alpha12:5
+	virtual/pkgconfig
+	test? ( dev-lang/python )
```

Comment:
I think this should be `dev-lang/python[${PYTHON_USEDEP}]`.

Reply:
This actually broke, replaced with ${PYTHON_DEPS} now